### PR TITLE
Reduce docker size

### PR DESCRIPTION
**Pull request by SimonHeybrock**
- Clean conda cache.
- Download files early, since they change rarely/never.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,7 +34,8 @@ RUN conda install --yes \
       scipp \
       ipywidgets \
       mantid-framework=4 \
-      python=3.6
+      python=3.6 && \
+      conda clean -afy
```
Comment (DanNixon) [inline Dockerfile]:
```suggestion
    conda clean -afy
```

File: Dockerfile [hunk 2]
```diff
@@ -3,19 +3,26 @@ FROM jupyter/base-notebook
 # Avoid using passwords for jupyter notebook
 USER root
 RUN apt-get update
-RUN apt-get install -y freeglut3-dev libglu1-mesa
+RUN apt-get install -y freeglut3-dev libglu1-mesa && \
```
Comment (DanNixon) [inline Dockerfile]:
```suggestion
RUN apt-get install -y libgl1 libglu1-mesa && \
```

Comment (SimonHeybrock) [reply Dockerfile]: Have you verified that this works?

Comment (DanNixon) [reply Dockerfile]: Yes

**Pull request merged**